general/development/policies/deprecation/index.md

```diff
@@ -124,49 +130,81 @@ The final deprecation policy for Moodle LMS has been updated to align more close
   - Functions that have been deprecated in an LTS version (the last version within a series) will be up for final deprecation on the next major version after the next LTS release (`[X+2].0`).
 
 <ValidExample title="Example">
-    - Functions deprecated in Moodle 4.4 (Series 4) and below will be up for final deprecation in Moodle 5.0 (the first Series 5 Moodle version).
+    - Functions deprecated in Moodle 4.4 (Series 4) and below will be considered for final deprecation in Moodle 5.0 (the first Series 5 Moodle version).
     - Functions deprecated in Moodle 4.5 (LTS) will be up for final deprecation in Moodle 6.0 (the first release for Series 6 right after the Moodle 5.3 (LTS) release).
 </ValidExample>
 
 #### Procedure
 
-- When a function undergoes final deprecation, all content of the function should be removed. In the skeleton that remains, an error statement should be included that indicates that the function cannot be used anymore. You can also direct developers to the new function(s) in this message.
+- When a function undergoes final deprecation, all content of the function should be removed. In the skeleton that remains, an error statement should be included that indicates that the function can not be used anymore. You can also direct developers to the new function(s) in this message.
```

Comment:
I think 'cannot' is more correct than 'can not' here. So, pity to see you changing it the 'wrong' way: https://www.google.com/search?client=firefox-b-e&channel=entpr&q=cannot+vs+can+not. (Sorry, grammar pedant.)

Reply:
More importantly "You can also direct developers to ...". I would say "can (and should)" just to re-inforce a point made elsewhere on this page.